### ISSUE TITLE
When Using Nav Menu, divs from each slider appear at the stacked

### ISSUE THREAD

**Issue by sjm375** [alvarotrigo/pagePiling.js#146]
Hello,

On my site:
http://sociallyin11.wpengine.com/

When I use the navigation menu to move down to one of the other sliders, the divs with the text at the bottom of each slider appear stacked over each other.

Is this a problem with not having a few lines of JQuery?

Let me know if you have seen this problem before.

Thanks

**Comment by alvarotrigo** [comment]
Sorry I don't understand what is the issue. Can you please express it in another way or detail what you would expect?

**Comment by sjm375** [comment]
Sorry for being unclear,

What I meant to say was that when using the navigation buttons on the right-hand side of the page, the text at the bottom is not disappearing like it is supposed to, each div from each slider is stacking on top of each other.

**Comment by alvarotrigo** [comment]
Right. Probably some kind of CSS issue, but as pagePiling.js doesn't provide that text, I don't believe it is directly related with pagePiling.js

Also, it doesn't always happen. Only sometimes.

**Comment by sjm375** [comment]
Thank you for the suggestion, I will look into the issue.
